src/main/java/com/github/sgov/server/controller/util/RestExceptionHandler.java

```diff
@@ -151,7 +152,17 @@ private static ErrorInfo errorInfo(HttpServletRequest request, Throwable e) {
     }
 
     /**
-     * Publication Exception.
+     * Feature Disabled Exception.
+     */
+    @ExceptionHandler(FeatureDisabledException.class)
+    public ResponseEntity<ErrorInfo> publicationException(HttpServletRequest request,
+                                                          FeatureDisabledException e) {
+        logException("Feature disabled exception caught.", e);
+        return new ResponseEntity<>(errorInfo(request, e), HttpStatus.CONFLICT);
```

Comment:
Not sure about 409 - in which sense is it "a request conflict with the current state of the target resource."?

Isn't 404 more appropriate, e.g.?

Reply:
Both 404 and 409 are not very good options. But after rethinking, it agree that 404 is more correct.

On the other hand, I would consider changing it to 409 if we implement funcionality to "enable publishing" by REST API. 

404 -- "location/resource not exists" -- to me resource disabled is different than resource does not exists and I wanted to indicate that location is correct, just disabled.
409 -- "a request conflict with the current state of the target resource." -- state of the target resource  is "disabled". 

I found similar line of reasoning here where they also suggest 409:
[Expected HTTP status code for an action on a disabled resource](https://stackoverflow.com/questions/36874263/expected-http-status-code-for-an-action-on-a-disabled-resource)

Interesting note by a user:
"""
That seems more like the entity shifted underneath the client while the client constructed a call, perhaps the assumption is the client asks for the allowed actions, before making the call(s). If you implemented that, I would say 409 is perfectly valid for your API since you provide the ability for clients to make only valid requests, unless someone else changes the entity.
"""

Relevant text in specification:
409 - 
"""
This code is only allowed in situations where it is expected that the **user might be able to resolve the conflict and resubmit the request**. The response body SHOULD include enough information for the user to recognize the source of the conflict. Ideally, the response entity would include enough information for the user or user agent to fix the problem; however, that might not be possible and **is not required**.
 """

404 - 
"""
The server has **not found anything matching the Request-URI**. No indication is given of whether the condition is temporary or permanent. The 410 (Gone) status code SHOULD be used if the server knows, through some internally configurable mechanism, that an old resource is permanently unavailable and has no forwarding address. **This status code is commonly used when the server does not wish to reveal exactly why the request has been refused, or when no other response is applicable.**
"""

I guess strongest argument against 409, is "user might be able to resolve the conflict and resubmit the request". If we don't have API for the user to enable the functionality we should use 404.

Reply:
Implemented (let me know if u change opinion to revert it ;).